docker/resource_docker_container.go

```diff
@@ -159,6 +159,11 @@ func resourceDockerContainer() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"workdir": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
We need to add `ForceNew: true` too here. If the user changes the `workdir`, the container needs to be recreated in order to match the desired configuration. (With this provider I think `ForceNew` is needed almost everywhere)